### PR TITLE
add an example for callback uri

### PR DESCRIPTION
**Pull request by satyakonmsft**

### CODE REVIEW

File: Samples/LocalMediaSamples/README - multiple app support.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ public ICall Call { get; }
...
 
-For example, if our service URI is `https://base.uri/callback`, we can set all our bots to use this URI directly, but then we lose knowledge of which app is receiving the incoming call.  If we change each apps callback uri to `https://base.uri/callback/{appId}` then we have the app context when an incoming call occurs.  Of course there are other ways at getting the app id, like from the auth token or the payload, but this is one simple option.
+For example, if our service URI is `https://base.uri/callback`, we can set all our bots to use this URI directly, but then we lose knowledge of which app is receiving the incoming call.  If we change each apps callback uri to `https://base.uri/callback/{appId}` (example: https://base.uri/callback/9ecd52e5-6592-42b7-b562-093f37f13bde, where the appId is 9ecd52e5-6592-42b7-b562-093f37f13bde) then we have the app context when an incoming call occurs.  Of course there are other ways at getting the app id, like from the auth token or the payload, but this is one simple option.
```
Comment (ksikorsk) [inline Samples/LocalMediaSamples/README - multiple app support.md]: Hey Satya, please make this change in our internal repo.  We overwrite this with the content there.

Comment (ksikorsk) [reply Samples/LocalMediaSamples/README - multiple app support.md]: I'll copy this change to the internal repo.

Comment (Lauragra) [inline Samples/LocalMediaSamples/README - multiple app support.md]: Change "apps callback uri" to "app's callback URI". Change "ways at getting the app id" to "ways to get the app ID".

Comment (satyakonmsft) [reply Samples/LocalMediaSamples/README - multiple app support.md]: changed it, thanks for the review

**Pull request merged**